clients/web/src/stylesheets/body/adminConsole.styl

```diff
@@ -2,7 +2,7 @@ ul.g-admin-options
   list-style-type none
   font-size 18px
   padding-left 0
-  maring 0
```

Comment:
No, this is a typo.

What you really want is `marring: false;`, telling the layout engine not to [disfigure](https://www.merriam-webster.com/dictionary/mar) your page and for CSS to always do exactly that you want. However, none of the user agents have implemented this feature yet.